tools/android-build

```diff
@@ -13,3 +13,8 @@ cp trustwalletcore/build/outputs/aar/trustwalletcore-release.aar ../build/trustw
 popd
 
 echo "Now upload build/trustwalletcore.aar to https://github.com/TrustWallet/trust-wallet-core/releases/tag/$version"
+
+echo "Building docs..."
+tools/kotlin-doc
+
+echo "Now upload Kotlin docs from build/kdoc.zip to whatever place it needs to be"
```

Comment:
let's add build and upload step to `tools/android-release` script.

Reply:
Take a look at how to upload it: https://github.com/trustwallet/wallet-core/blob/master/tools/ios-xcframework-release#L85